changelog/tidy-remove-helix-tracks-names-constants

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Refactor dispute & deposits tracks event names inline. `wcpay_disputes_row_action_click` now also fires from row/cell clicks, not just the action button.
```

Comment:
⛏️ I would consider making this a "Comment:" placeholder changelog entry, since it has no user-facing changes.

Or, at least, the `Refactor dispute & deposits tracks event names inline. ` part is not user-facing.